tests/scientific/test_replay.py

```diff
@@ -43,7 +43,7 @@ def test_replay_sim(sonata_config):
     )
 
     node_managers = nd.circuits.node_managers
-    assert set(node_managers) == set([None, "NodeA"])
+    assert set(node_managers) == set(["NodeA"])
```

Comment:
since there is only one, no need for `set` anymore?

Reply:
I suspect the brackets are superfluous. If you want to maintain the set to set comparison:
```suggestion
    assert set(node_managers) == {"NodeA"}
```

Reply:
Applied for all the occurrences in this file